.github/workflows/superset-python.yml

```diff
@@ -29,6 +29,25 @@ jobs:
       # `-j 0` run Pylint in parallel
       run: pylint -j 0 superset
 
+  docs:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.6]
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      uses: apache-superset/cached-dependencies@adc6f73
```

Comment:
Can move `pip install -r docs/requirements.txt` here:

```
    - name: Install dependencies
      uses: apache-superset/cached-dependencies@adc6f73
      with:
        run: |
          pip-install
          pip install -r docs/requirements.txt
```

Probably save a couple of seconds for other jobs.

Reply:
Done.